config/config.go

```diff
@@ -84,5 +89,10 @@ func (d DatadogConfig) GetApmEndpoint() string {
 
 // IsExtraProfilingEnabled return true if profilers not recommended for production are enabled.
 func (d DatadogConfig) IsExtraProfilingEnabled() bool {
-	return d.EnableExtraProfiling
+	isEnabled, failedToConvert := strconv.ParseBool(strings.ToLower(strings.TrimSpace(d.EnableExtraProfiling)))
```

Comment:
`strconv.ParseBool` accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.
ref: https://pkg.go.dev/strconv#ParseBool

So you don't really need to use `strings.ToLower`

Reply:
Yeah they do that inside anyway